src/pconn.cc

```diff
@@ -575,7 +575,8 @@ PconnModule::registerWithCacheManager(void)
 {
     Mgr::RegisterAction("pconn",
                         "Persistent Connection Utilization Histograms",
-                        DumpWrapper, 0, 1);
+                        DumpWrapper, Mgr::Protected::no, Mgr::Atomic::yes,
+                        Mgr::Format::yaml);
```

Comment:
This is the only `mgr:pconn` change. Everything else is scaffolding to enable this and future YAML-compliant reporting.